### DISCUSSION TITLE
Porting code written for an older version of ChampSim

### DISCUSSION THREAD

**Discussion by afschy** [ChampSim/ChampSim#496] [Q&A]
I am trying to port some code written for an older version of ChampSim (around 2019) to the current version. The older code has get_occupancy(3, 0) and get_size(3, 0) which are supposed to return the occupancy and the size of the PQ as size_t. What are the function calls that do the same thing given that both get_occupancy and get_size have been deprecated and no longer work? I tried to use get_pq_occupancy and get_pq_size but they both return vectors whereas the old code expects a single integer value.

**Comment by maccoymerrell** [comment]
Looking at the code, index 0 of those vectors should contain your desired value. They will be size_t types.
